app/routes/_index.tsx

```diff
@@ -1,16 +1,25 @@
-import {defer, type LoaderFunctionArgs} from '@netlify/remix-runtime';
+import {
+  defer,
+  type HeadersFunction,
+  type LoaderFunctionArgs,
+} from '@netlify/remix-runtime';
 import {Await, useLoaderData, Link, type MetaFunction} from '@remix-run/react';
 import {Suspense} from 'react';
 import {Image, Money} from '@shopify/hydrogen';
 import type {
   FeaturedCollectionFragment,
   RecommendedProductsQuery,
 } from 'storefrontapi.generated';
+import {CACHE_1_HOUR_SWR} from '~/lib/page-cache';
 
 export const meta: MetaFunction = () => {
   return [{title: 'Hydrogen | Home'}];
 };
 
+export const headers: HeadersFunction = () => ({
```

Comment:
https://remix.run/docs/en/main/route/headers

This is inherited by all child routes, unless a child route (or... closer ancestor parent route) overrides it.